roku/core.py

```diff
@@ -149,7 +149,7 @@ def discover(self, *args, **kwargs):
         return rokus
 
     def __init__(self, host, port=8060, timeout=10):
-        self.host = host
+        self.host = socket.gethostbyname(host)
```

Comment:
This is great, as long as we don't use IPv6 addresses. IPv6 will break this function. This is not currently an issue, as Roku only supports IPv4 at this time, but worth noting that it will break forward compatibility when this changes. There is currently no public timeline for IPv6 support from Roku, however.

Reply:
Agreed. IPv6 has been the can kicked down the road. Eventually, this will have to be redone, but for now, I think its a worthy addition. FWIW, I've been running this change on my system (via Home Assistant) and its been working just fine for about a week now.